packages/astro/e2e/actions-blog.test.js

```diff
@@ -135,4 +135,17 @@ test.describe('Astro Actions - Blog', () => {
 		await logoutButton.click();
 		await expect(page).toHaveURL(astro.resolveUrl('/blog/'));
 	});
+
+	test('Should redirect to the origin pathname when there is a rewrite', async ({
+		page,
+		astro,
+	}) => {
+		await page.goto(astro.resolveUrl('/sum'));
+		//
```

Comment:
```suggestion
		await page.goto(astro.resolveUrl('/sum'));
```

we could remove this stray comment i think